shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx

```diff
@@ -0,0 +1,119 @@
+const React = require('react');
+const { DocketHeader } = require('../components/DocketHeader.jsx');
+const { PrimaryHeader } = require('../components/PrimaryHeader.jsx');
+
+export const StandingPretrialNotice = ({ options, trialInfo }) => {
+  return (
+    <>
+      <PrimaryHeader />
+      <DocketHeader
+        caseCaptionExtension={options.caseCaptionExtension}
+        caseTitle={options.caseTitle}
+        docketNumberWithSuffix={options.docketNumberWithSuffix}
+        h3="Standing Pre Trial Notice"
```

Comment:
I think `Pretrial` is one word here

Reply:
I messed up my local staging branch so this file was erroneously included. I deleted it to match staging.

Although to be fair, I wasn't the one who misspelled this 😁